src/test/java/uk/gov/hmcts/cmc/claimstore/utils/ResourceLoader.java

```diff
@@ -43,6 +43,11 @@ public static CaseDetails successfulCoreCaseDataStoreSubmitResponse() {
         return jsonMapper.fromJson(response, CaseDetails.class);
     }
 
+    public static CaseDetails successfulCoreCaseDataStoreSubmitResponseWithDQ() {
+        String response = getResource("/core-case-data/submit-response.success-with-dq.json");
```

Comment:
Curiosity - Why `.` between words in file name?

Reply:
was just following the existing patter 🤷‍♀